tests/autodiff/let-var-differentiable-initializer.slang.expected.txt

```diff
@@ -0,0 +1,6 @@
+type: float
```

Comment:
Use file check instead of a separate expected.txt file. See `tests\spirv\bool-vector.slang` for an example.